support-frontend/assets/helpers/abTests/abtest.js

```diff
@@ -201,6 +202,14 @@ function getParticipations(
       return;
     }
 
+    if (test.urlParticipationOnly) {
+      const participationsFromUrl = getParticipationsFromUrl();
+      if (participationsFromUrl && participationsFromUrl[testId]) {
+        participations[testId] = participationsFromUrl[testId];
+      }
+      return;
```

Comment:
can we just use an `else` instead of returning half way through?  then the structure seems a bit more level